README.md

```diff
@@ -38,17 +38,13 @@ You can use the simple python API to push your metrics to your storage. First, m
 ```python
 import cimetrics.upload
 
-metrics = cimetrics.upload.Metrics()
-
-# Run some tests and collect some data
-
-metrics.put("metric1 name (unit)", metric_1)
-metrics.put("metric2 name (unit)", metric_2)
-
-metrics.publish()
+with cimetrics.upload.Metrics() as metrics:
+  # Run some tests and collect some data
+  metrics.put("metric1 name (unit)", metric_1)
```

Comment:
We should show the "^" behaviour here.

Reply:
I'm hoping to kill that soon and make it a property, along with the category (unit/end-to-end), that's why I've not mentioned it here.